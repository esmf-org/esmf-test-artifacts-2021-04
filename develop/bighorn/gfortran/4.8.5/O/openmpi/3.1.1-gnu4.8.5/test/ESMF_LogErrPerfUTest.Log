build time -- 2021-04-28 10:56:36.496283
20210428 110433.515 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210428 110433.515 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210428 110433.515 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210428 110433.515 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210428 110433.515 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210428 110433.515 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210428 110433.515 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-6-gb8c1a7077f
20210428 110433.515 INFO             PET0 ESMF library build date/time: "Apr 28 2021" "10:55:16"
20210428 110433.515 INFO             PET0 ESMF library build location : /scratch/mpotts/gfortran_4.8.5_openmpi_O_develop
20210428 110433.515 INFO             PET0 ESMF_COMM                   : openmpi
20210428 110433.515 INFO             PET0 ESMF_MOAB                   : enabled
20210428 110433.515 INFO             PET0 ESMF_LAPACK                 : enabled
20210428 110433.515 INFO             PET0 ESMF_NETCDF                 : enabled
20210428 110433.515 INFO             PET0 ESMF_PNETCDF                : disabled
20210428 110433.515 INFO             PET0 ESMF_PIO                    : enabled
20210428 110433.515 INFO             PET0 ESMF_YAMLCPP                : enabled
20210428 110433.516 INFO             PET0  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20210428 110433.516 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210428 110433.516 INFO             PET0  perfLogFoundError:         1000  iterations took   1.0900665074586868E-005  seconds. =>    1.0900665074586868E-008  per iteration.
20210428 110433.516 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20210428 110433.516 INFO             PET0  perfLogFoundError:        10000  iterations took   1.0035093873739243E-004  seconds. =>    1.0035093873739243E-008  per iteration.
20210428 110433.516 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20210428 110433.517 INFO             PET0  perfLogFoundError:       100000  iterations took   1.0061752982437611E-003  seconds. =>    1.0061752982437611E-008  per iteration.
20210428 110433.517 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20210428 110433.527 INFO             PET0  perfLogFoundError:      1000000  iterations took   9.9353217519819736E-003  seconds. =>    9.9353217519819728E-009  per iteration.
20210428 110433.527 INFO             PET0  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20210428 110433.527 INFO             PET0  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
20210428 110433.527 INFO             PET0  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
20210428 110433.527 INFO             PET0 Finalizing ESMF
20210428 110433.515 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210428 110433.515 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210428 110433.515 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210428 110433.515 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210428 110433.515 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210428 110433.515 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210428 110433.515 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-6-gb8c1a7077f
20210428 110433.515 INFO             PET1 ESMF library build date/time: "Apr 28 2021" "10:55:16"
20210428 110433.515 INFO             PET1 ESMF library build location : /scratch/mpotts/gfortran_4.8.5_openmpi_O_develop
20210428 110433.515 INFO             PET1 ESMF_COMM                   : openmpi
20210428 110433.515 INFO             PET1 ESMF_MOAB                   : enabled
20210428 110433.515 INFO             PET1 ESMF_LAPACK                 : enabled
20210428 110433.515 INFO             PET1 ESMF_NETCDF                 : enabled
20210428 110433.515 INFO             PET1 ESMF_PNETCDF                : disabled
20210428 110433.515 INFO             PET1 ESMF_PIO                    : enabled
20210428 110433.515 INFO             PET1 ESMF_YAMLCPP                : enabled
20210428 110433.516 INFO             PET1  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20210428 110433.516 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210428 110433.516 INFO             PET1  perfLogFoundError:         1000  iterations took   1.0543037205934525E-005  seconds. =>    1.0543037205934524E-008  per iteration.
20210428 110433.516 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20210428 110433.516 INFO             PET1  perfLogFoundError:        10000  iterations took   9.6010975539684296E-005  seconds. =>    9.6010975539684303E-009  per iteration.
20210428 110433.516 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20210428 110433.517 INFO             PET1  perfLogFoundError:       100000  iterations took   9.7141927108168602E-004  seconds. =>    9.7141927108168609E-009  per iteration.
20210428 110433.517 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20210428 110433.526 INFO             PET1  perfLogFoundError:      1000000  iterations took   9.4660050235688686E-003  seconds. =>    9.4660050235688689E-009  per iteration.
20210428 110433.526 INFO             PET1  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20210428 110433.527 INFO             PET1  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
20210428 110433.527 INFO             PET1  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
20210428 110433.527 INFO             PET1 Finalizing ESMF
20210428 110433.515 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210428 110433.515 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210428 110433.515 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210428 110433.515 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210428 110433.515 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210428 110433.515 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210428 110433.515 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-6-gb8c1a7077f
20210428 110433.515 INFO             PET2 ESMF library build date/time: "Apr 28 2021" "10:55:16"
20210428 110433.515 INFO             PET2 ESMF library build location : /scratch/mpotts/gfortran_4.8.5_openmpi_O_develop
20210428 110433.515 INFO             PET2 ESMF_COMM                   : openmpi
20210428 110433.515 INFO             PET2 ESMF_MOAB                   : enabled
20210428 110433.515 INFO             PET2 ESMF_LAPACK                 : enabled
20210428 110433.515 INFO             PET2 ESMF_NETCDF                 : enabled
20210428 110433.515 INFO             PET2 ESMF_PNETCDF                : disabled
20210428 110433.515 INFO             PET2 ESMF_PIO                    : enabled
20210428 110433.515 INFO             PET2 ESMF_YAMLCPP                : enabled
20210428 110433.516 INFO             PET2  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20210428 110433.516 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210428 110433.516 INFO             PET2  perfLogFoundError:         1000  iterations took   1.0872725397348404E-005  seconds. =>    1.0872725397348404E-008  per iteration.
20210428 110433.516 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20210428 110433.516 INFO             PET2  perfLogFoundError:        10000  iterations took   1.0034022852778435E-004  seconds. =>    1.0034022852778434E-008  per iteration.
20210428 110433.516 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20210428 110433.517 INFO             PET2  perfLogFoundError:       100000  iterations took   1.0062870569527149E-003  seconds. =>    1.0062870569527150E-008  per iteration.
20210428 110433.517 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20210428 110433.527 INFO             PET2  perfLogFoundError:      1000000  iterations took   9.9092731252312660E-003  seconds. =>    9.9092731252312659E-009  per iteration.
20210428 110433.527 INFO             PET2  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20210428 110433.527 INFO             PET2  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
20210428 110433.527 INFO             PET2  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
20210428 110433.527 INFO             PET2 Finalizing ESMF
20210428 110433.515 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210428 110433.515 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210428 110433.515 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210428 110433.515 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210428 110433.515 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210428 110433.515 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210428 110433.515 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-6-gb8c1a7077f
20210428 110433.515 INFO             PET3 ESMF library build date/time: "Apr 28 2021" "10:55:16"
20210428 110433.515 INFO             PET3 ESMF library build location : /scratch/mpotts/gfortran_4.8.5_openmpi_O_develop
20210428 110433.515 INFO             PET3 ESMF_COMM                   : openmpi
20210428 110433.515 INFO             PET3 ESMF_MOAB                   : enabled
20210428 110433.515 INFO             PET3 ESMF_LAPACK                 : enabled
20210428 110433.515 INFO             PET3 ESMF_NETCDF                 : enabled
20210428 110433.515 INFO             PET3 ESMF_PNETCDF                : disabled
20210428 110433.515 INFO             PET3 ESMF_PIO                    : enabled
20210428 110433.515 INFO             PET3 ESMF_YAMLCPP                : enabled
20210428 110433.516 INFO             PET3  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
20210428 110433.516 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210428 110433.516 INFO             PET3  perfLogFoundError:         1000  iterations took   1.0527670383453369E-005  seconds. =>    1.0527670383453369E-008  per iteration.
20210428 110433.516 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
20210428 110433.516 INFO             PET3  perfLogFoundError:        10000  iterations took   9.6017960458993912E-005  seconds. =>    9.6017960458993920E-009  per iteration.
20210428 110433.516 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
20210428 110433.517 INFO             PET3  perfLogFoundError:       100000  iterations took   9.7103416919708252E-004  seconds. =>    9.7103416919708245E-009  per iteration.
20210428 110433.517 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
20210428 110433.526 INFO             PET3  perfLogFoundError:      1000000  iterations took   9.4757829792797565E-003  seconds. =>    9.4757829792797562E-009  per iteration.
20210428 110433.527 INFO             PET3  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
20210428 110433.527 INFO             PET3  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
20210428 110433.527 INFO             PET3  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
20210428 110433.527 INFO             PET3 Finalizing ESMF
