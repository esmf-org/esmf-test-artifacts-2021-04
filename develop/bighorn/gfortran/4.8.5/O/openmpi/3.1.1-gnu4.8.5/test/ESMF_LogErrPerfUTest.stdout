build time -- 2021-04-28 10:56:36.496283
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_LogErrPerfUTest.F90, line 55
  NUMBER_OF_PROCESSORS           4
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 1000x Test, ESMF_LogErrPerfUTest.F90, line 64
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 10000x Test, ESMF_LogErrPerfUTest.F90, line 72
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 100000x Test, ESMF_LogErrPerfUTest.F90, line 80
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
  PASS  Performance of ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 88
  PASS  Threshold check for ESMF_LogFoundError() 1000000x Test, ESMF_LogErrPerfUTest.F90, line 99
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
  Ending Test, file ESMF_LogErrPerfUTest.F90, line 103
  PET           3  Test Elapsed Time    15.845999999999998       msec.
  PET           0  Test Elapsed Time    16.392000000000003       msec.
  PET           2  Test Elapsed Time    16.454000000000004       msec.
  PET           1  Test Elapsed Time    15.997000000000003       msec.
